ui/redux/reducers/comments.js

```diff
@@ -1070,10 +1070,22 @@ export default handleActions(
       ...state,
       fetchingSettings: true,
     }),
-    [ACTIONS.COMMENT_FETCH_SETTINGS_FAILED]: (state: CommentsState, action: any) => ({
-      ...state,
-      fetchingSettings: false,
-    }),
+
+    [ACTIONS.COMMENT_FETCH_SETTINGS_FAILED]: (state: CommentsState, action: any) => {
+      const { channelId } = action.data;
+      const settingsByChannelId = Object.assign({}, state.settingsByChannelId);
```

Comment:
Isn't there a case that this could be null (no state previously loaded for channel) and for some reason the channel settings fetch failed - won't that lead to data loss?

Reply:
Also, why does this only happen with 0 views? Once you view something on the channel, then things work. Does it somewhere along the lines set default settings?

Reply:
The core issues seems to be on commentron, can try to make something there. Check the screenshot in the PR.  

I can also just simplify the fix and remove this check, and not do any other changes. Not really sure why it's in there. `if (settingsByChannelId && Object.keys(settingsByChannelId).length)` 

On the PR  I changed it to this:
`if (settingsByChannelId && Object.keys(settingsByChannelId).includes(claim.claim_id))`
This ensures that `setHome()` is only run once, with either channel's settings, or with the default settings. Current Odysee runs it twice, but so fast that I don't think this really makes any actual difference.)

--------------------------------------------
I'm not sure I understand the data loss issue.

Reply:
Maybe it's `comment.List` that fixes it, and not the view.  Will try to find some channel with the issue. 
![2024-05-17_17-21](https://github.com/OdyseeTeam/odysee-frontend/assets/34790748/4354765d-0274-41d8-a72f-2c9594c37490)

Reply:
Ah, it could be because there's no channel created on commentron at first, and listing does create one. Could be worth a fix there for settings get - create channel if doesn't exist. But sounds like this fix is solid anyhow.